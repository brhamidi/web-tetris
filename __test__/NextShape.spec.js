import React from 'react';
import { shallow } from 'enzyme';
import NextShape from '../src/client/components/nextShape';

import { List } from 'immutable'

describe('NextShape', () => {
	test('renders correctly', () => {
		const shape = List().set(3, undefined).map(e => List().set(3, undefined));
		const wrapper = shallow(<NextShape shape={shape} />);

		expect(wrapper).toMatchSnapshot();
	});
});
