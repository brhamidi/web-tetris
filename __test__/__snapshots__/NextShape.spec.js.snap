// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NextShape renders correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "black",
      "display": "flex",
      "flexFlow": "row wrap",
      "justifyContent": "center",
      "width": "400px",
    }
  }
>
  <h2
    style={
      Object {
        "color": "white",
        "textAlign": "center",
        "width": "300px",
      }
    }
  >
     Next Shape 
  </h2>
  <div
    style={
      Object {
        "display": "flex",
        "flexFlow": "row wrap",
        "height": "200px",
        "justifyContent": "center",
        "width": "200px",
      }
    }
  >
    <div
      key="00"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="01"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="02"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="03"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="10"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="11"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="12"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="13"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="20"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="21"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="22"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="23"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="30"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="31"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="32"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
    <div
      key="33"
      style={
        Object {
          "backgroundColor": "black",
          "height": "50px",
          "width": "50px",
        }
      }
    />
  </div>
</div>
`;
